test/testMethods.ts

```diff
@@ -353,15 +353,30 @@ module TestMethods {
   export function assertLinePathEqualToDataPoints(path: string, data: {x: number, y: number}[],
     xScale: Plottable.Scales.Linear, yScale: Plottable.Scales.Linear) {
     let EPSILON = 0.0001;
-    let lineEdges = TestMethods.normalizePath(path).match(/(\-?\d+\.?\d*)(,|\s)(-?\d+\.?\d*)/g);
+    let lineEdges = TestMethods.normalizePath(path).match(/([MmLl](\-?\d+\.?\d*)(,|\s)(-?\d+\.?\d*)|([HhVv](\-?\d+\.?\d*)))/g);
```

Comment:
moveto(Mm) and lineto(Ll) are followed by (x, y) and horizontal lineto(Hh) and vertical lineto (Vv) are followed by only one coordinate
for step functions, horizontal/vertical lineto are used, so I changed the reg exp to consider those cases too